bin/htmlproof

```diff
@@ -29,31 +29,31 @@ Mercenary.program(:htmlproof) do |p|
   p.option 'check_external_hash', '--check_external_hash', 'Checks whether external hashes exist (even if the website exists). This slows the checker down (default: `false`).'
 
   p.action do |args, opts|
-    args = ["."] if args.empty?
+    args = ['.'] if args.empty?
     path = args.first
 
     options = {}
-    options[:ext] = opts["ext"] unless opts["ext"].nil?
-    unless opts["swap"].nil?
+    options[:ext] = opts['ext'] unless opts['ext'].nil?
+    unless opts['swap'].nil?
       options[:href_swap] = {}
-      opts["swap"].each do |s|
-        pair = s.split(":")
-        options[:href_swap][%r{#{pair[0]}}] = pair[1]
+      opts['swap'].each do |s|
+        pair = s.split(':')
+        options[:href_swap][/#{pair[0]}/] = pair[1]
       end
     end
 
-    options[:href_ignore] = opts["href_ignore"] unless opts["href_ignore"].nil?
-    options[:alt_ignore] = opts["alt_ignore"] unless opts["alt_ignore"].nil?
-    options[:file_ignore] = opts["file_ignore"] unless opts["file_ignore"].nil?
-    options[:disable_external] = opts["disable_external"] unless opts["disable_external"].nil?
-    options[:only_4xx] = opts["only_4xx"] unless opts["only_4xx"].nil?
-    options[:favicon] = opts["favicon"] unless opts["favicon"].nil?
-    options[:verbose] = opts["verbose"] unless opts["verbose"].nil?
-    options[:directory_index_file] = opts["directory_index_file"] unless opts["directory_index_file"].nil?
-    options[:validate_html] = opts["validate_html"] unless opts["validate_html"].nil?
-    options[:check_external_hash] = opts["check_external_hash"] unless opts["check_external_hash"].nil?
-
-    path = path.delete(' ').split(",") if opts["as-links"]
+    options[:href_ignore] = opts['href_ignore'] unless opts['href_ignore'].nil?
+    options[:alt_ignore] = opts['alt_ignore'] unless opts['alt_ignore'].nil?
+    options[:file_ignore] = opts['file_ignore'] unless opts['file_ignore'].nil?
+    options[:disable_external] = opts['disable_external'] unless opts['disable_external'].nil?
+    options[:only_4xx] = opts['only_4xx'] unless opts['only_4xx'].nil?
+    options[:favicon] = opts['favicon'] unless opts['favicon'].nil?
+    options[:verbose] = opts['verbose'] unless opts['verbose'].nil?
+    options[:directory_index_file] = opts['directory_index_file'] unless opts['directory_index_file'].nil?
+    options[:validate_html] = opts['validate_html'] unless opts['validate_html'].nil?
+    options[:check_external_hash] = opts['check_external_hash'] unless opts['check_external_hash'].nil?
```

Comment:
ugh probably. I was thinking about removing the bin entirely but I guess that's Not The Right Thing To Do :tm:.